### PR TITLE
When using the --password env:FOO will now return an errors if

### PR DESCRIPTION
**Pull request by ph**
FOO does not exist in the environment.

### CODE REVIEW

File: libbeat/common/cli/password.go [hunk 1]
```diff
@@ -75,5 +75,10 @@ func env(p string) (string, error) {
 		return "", errors.New("env variable name is needed when using env: password method")
```
Comment (andrewkroh) [inline libbeat/common/cli/password.go]: I noticed the error uses inconsistent terminology "env var" vs "environment variable". I prefer the latter like what you have added in the new error.

Comment (ph) [reply libbeat/common/cli/password.go]: good point, I will replace that.

**Comment by ph** [comment]
jenkins test this please

**Comment by ph** [comment]
metricbeat error is not related.

**Pull request merged**